### PR TITLE
feat: Add a setting for elements bounding strategy selection

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Adds `boundElementsByIndex` boolean setting, which allows to select the bounding strategy for elements lookup. Based on https://github.com/appium/appium/issues/14486

### CODE REVIEW

File: WebDriverAgentLib/Utilities/FBConfiguration.h [hunk 1]
```diff
@@ -181,6 +181,17 @@ typedef NS_ENUM(NSInteger, FBConfigurationKeyboardPreference) {
 + (void)setUseFirstMatch:(BOOL)enabled;
 + (BOOL)useFirstMatch;
 
+/**
+ * Whether to bound the lookup results by index.
+ * By default this is disabled and bounding by accessibility is used.
+ * Read https://stackoverflow.com/questions/49307513/meaning-of-allelementsboundbyaccessibilityelement
```
Comment (KazuCocoa) [inline WebDriverAgentLib/Utilities/FBConfiguration.h:187]: mm, i see..

**Review by KazuCocoa — APPROVED** [review]

**Comment by vitalyk** [comment]
Hello @mykola-mokhnach, could you please clarify where exactly `boundElementsByIndex` boolean setting for the iOS should be specified in case of the following error:
```
 the previously found element ""currency" StaticText" is not present in the current view anymore. Make sure the application UI has the expected state. You could also try to switch the binding strategy using the 'boundElementsByIndex' setting for the element lookup. Original error: No matches found for Identity Binding from input
```
Thanks ahead

**Comment by KazuCocoa** [comment]
https://appium.io/docs/en/advanced-concepts/settings/

**Comment by vitalyk** [comment]
Thanks @KazuCocoa for your reply.
Sorry, it’s little bit unclear- should I add this particular setting within capabilities? e.g:
```
{ caps: { automationName: 'XCUITest',
              platformName: 'iOS',
              ...,
              boundElementsByIndex: true }
}
```
Or this is something different?
Thanks a lot for your reply.

**Comment by KazuCocoa** [comment]
An API https://appium.io/docs/en/commands/session/settings/update-settings/index.html helps to set the settings. (Then, you can configure this value as you want)
You also can set it in capabilities as https://appium.io/docs/en/writing-running-appium/caps/#update-settings . Then, appium will call the settings API internally as part of a new session request.

**Pull request merged**